### PR TITLE
[docs] Categorize Integrations

### PR DESCRIPTION
**Pull request by ddhodge**
Put various Integrations into categories

@netlify /preview/integrations/

### CODE REVIEW

File: docs/content/preview/drivers-orms/include-drivers-orms-list.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+| Django     | | Full | [CRUD](/preview/drivers-orms/python/django/) |
+| SQLAlchemy | | Full | [CRUD](/preview/drivers-orms/python/sqlalchemy/) |
+
+  {{% /tab %}}
+
+  {{% tab header="NodeJS" lang="nodejs" %}}
+
+| Driver/ORM | Version | Support Level | Example apps |
+| :--------- | :------ | :------------ | :----------- |
+| YugabyteDB node-postgres Smart Driver [Recommended] | [8.7.3-yb-1](https://www.npmjs.com/package/pg) | Full | [CRUD](/preview/drivers-orms/nodejs/yugabyte-node-driver/) |
```
Comment (aishwarya24) [inline docs/content/preview/drivers-orms/include-drivers-orms-list.md:57]: [8.7.3-yb-1](https://www.npmjs.com/package/pg) - Doesn't look like the right link.

**Review by aishwarya24 — APPROVED** [review]
some comments.

**Comment by ymahajan** [comment]
looks good, Presto can be moved to 'other' it's not really a metadata category.





[12:45](https://yugabyte.slack.com/archives/D022UA48L8N/p1696362301351999)
Also is there a way to distinguish drivers from ORMs? worst case we can add an additional column to specify it's ORM or Driver.

**Comment by ymahajan** [comment]
It looks good overall, Presto can be moved to 'other' it's not really a metadata category.

Also is there a way to distinguish drivers from ORMs? worst case we can add an additional column to specify it's ORM or Driver.

**Review by ymahajan — APPROVED** [review]

**Pull request merged**